samples/drivers/led_ws2812/sample.yaml

```diff
@@ -3,5 +3,7 @@ sample:
   name: WS2812 sample
 tests:
   sample.drivers.led.ws2812:
-    platform_allow: bbc_microbit nrf51dk_nrf51422 nrf52dk_nrf52832
+    platform_allow: |
+      bbc_microbit nrf51dk_nrf51422 nrf52dk_nrf52832 mimxrt1050_evk
+      mimxrt1050_evk_qspi
```

Comment:
Agreed. Fixed.

Reply:
please adjust 2nd commit message now

Reply:
Updated

Reply:
it still doesn't reflect the change, maybe `led_ws2812 sample: filter based on compatible`

Reply:
Do you want me to place it in the commit message subject line or change description?

Reply:
both need to be consistent, and explain what the commit does

Reply:
I cannot fit both changes into a subject less than 72 characters. I'll put it in a separate commit if that is alright with you?

Reply:
something like this should be ok:

```
samples: drivers: led_ws2812: use compatible based filter

Replace 'platform_allow' with 'filter' based on ws2812 compatible
being enabled. This will allow, for example, to run the sample on
mimxrt1050_evk and mimxrt1050_evk_qspi boards.
```

Reply:
I was a bit too quick with the git commands and thus closed this PR by a mistake.
Sorry about that.
I have opened a new PR here: https://github.com/zephyrproject-rtos/zephyr/pull/46936

Reply:
there's no need to open new PRs, you can force-push here